addons/website_event/static/src/js/website_event_ticket_details.js

```diff
@@ -38,9 +38,9 @@ odoo.define('website_event.ticket_details', function (require) {
         _onTicketDetailsClick: function (ev){
             ev.preventDefault();
             if (this.foldedByDefault){
-                $(ev.currentTarget).toggleClass('btn-primary text-left pl-0');
-                $(ev.currentTarget).siblings().toggleClass('d-none');
-                this.$('.close').toggleClass('d-none');
+                $(ev.currentTarget).toggleClass('btn-primary');
+                $(ev.currentTarget).siblings('.o_toggle-d-none').toggleClass('d-none');
```

Comment:
I'm not a fan of that magic ".o_toggle-d-none" class.
It makes the code a bit cryptic.

At this point we are better of specifically targeting the things we want to show/hide by giving them a specific class.
(While you're at it, this method could be a bit documented to explain what it does too).

Reply:
The method now targets directly impacted elements.
Comments have been added on the JS.